### PR TITLE
Fix menu imports

### PR DESCRIPTION
**Pull request by emil-furniss**
The core menu components were [recently unbundled](https://github.com/Brightspace/brightspace-integration/pull/2435), and we were not explicitly importing them so the menu components in the grades dropdown were not being rendered properly.

### CODE REVIEW

**Review by wongvincent — APPROVED** [review]

**Review by wongvincent — COMMENTED** [review]
There are other places where the dropdown-menu is being used, are there going to be other PRs to fix those?

**Comment by emil-furniss** [comment]
@wongvincent Importing them in one place actually fixes all locations.

**Review by mostlyfabulous — DISMISSED** [review]

**Comment by mostlyfabulous** [comment]
Good find @emil-furniss

**Comment by emil-furniss** [comment]
Decided to import them in the other locations that might need it as well. Technically a single import within the page will do, but if that component is not loaded (for whatever reason) then the others would be in trouble.

File: components/d2l-activity-editor/d2l-activity-conditions-editor.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import '@brightspace-ui/core/components/colors/colors.js';
 import '@brightspace-ui/core/components/icons/icon.js';
 import 'd2l-dropdown/d2l-dropdown-button-subtle.js';
 import 'd2l-dropdown/d2l-dropdown-menu.js';
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-conditions-editor.js:4]: think you need to replace these

Comment (emil-furniss) [reply components/d2l-activity-editor/d2l-activity-conditions-editor.js:4]: Nah, technically not. Replaced them in the other location out of habit, but the dropdown imports are not the problem.

Comment (emil-furniss) [reply components/d2l-activity-editor/d2l-activity-conditions-editor.js:4]: It's one of the legacy imports that technically still work. We could go around and redirect all of them at the new core versions, but I think that's out of scope for this.

**Review by wongvincent — APPROVED** [review]

File: components/d2l-activity-editor/d2l-activity-conditions-editor.js (hunk 1 above)
Comment (dbatiste) [reply components/d2l-activity-editor/d2l-activity-conditions-editor.js:4]: The old legacy imports (referenced here) were updated to point to core when we did the migration, which is why they still work, but it results in an extra HTTP request in the unbundled scenario.  We definitely would like to see this switched eventually.

**Pull request merged**